### DISCUSSION TITLE
Export specific rows from table

### DISCUSSION THREAD

**Discussion by CynthiaBlue** [dexie/Dexie.js#2100] [Q&A]
I need to export specific rows from tables.  I'm able to export an entire table from the database

```
const getStuffToExport = await exportDB(pawTapDB, {
        prettyJson: true, 
       filter: (table, value, key) => table === "trialTable", 
})

download(getStuffToExport, "theBackupFile.json", "application/json")

```

But I cannot seem to specify which rows in the table I want.  Here is a snippet of my JSON exported file:

```
{

      "tableName": "trialTable",

      "inbound": true,

      "rows": [

        {

          "fk_event": 1,

          "trialNum": 1,

          "trialDate": "2022-06-04",

          "trialEventNumber": "2022745113",

          "buried": {

            "buriedNoviceOffered": true,

            "buriedNoviceJudge": 3,
```

I need to be able to export everying in the trialTable where the fk_event = 1 (or a user selection)

Is this possible?

In the filter, it has three things.  table, value, key.  I understand that table is the table name.  But what are the value and the key?  Can I use those for my selections?  I don't understand from reaading the documentation.

[https://dexie.org/docs/ExportImport/dexie-export-import#exportoptions](url)

Thanks.

**Comment by dfahlander** [comment]
Duplicate of https://stackoverflow.com/questions/79218784/use-dexie-to-export-certain-data-from-a-table-in-a-database
